auxx/src/AuxxOptions.hs

```diff
@@ -71,25 +72,30 @@ cmdParser = command "cmd" $ info opts desc
 -- Parse everything
 ----------------------------------------------------------------------------
 
+helpText :: Doc
+helpText = fromString @Doc $ toString @Text $ [N.text|
+    Auxx start mode. Can be 'light', 'with-config', 'with-node', 'auto'.
+    * 'light' mode requires no configuration to start, but the available.
+      command set is limited.
+    * 'with-node' mode will start auxx as a node plugin.
+    * 'with-config' mode will start auxx as a standalone application.
+    * 'auto' will try to load the configuration and start in 'with-config' mode,
+      if fails to do so will start in 'light' mode. (default: auto)
+    |]
+
 startModeParser :: Parser AuxxStartMode
-startModeParser = option startModeReader $
-    long "mode" <>
-    value Automatic <>
-    helpDoc (Just $
-           text "Auxx start mode. Can be 'light', 'with-config', 'with-node', 'auto'." <> line
-        <> text "* 'light' mode requires no configuration to start, but the available."
-        <> text " command set is limited." <> line
-        <> text "* 'with-node' mode will start auxx as a node plugin." <> line
-        <> text "* 'with-config' mode will start auxx as a standalone application." <> line
-        <> text "* 'auto' will try to load the configuration and start in 'with-config' mode,"
-        <> text " if fails to do so will start in 'light' mode. (default: auto)")
+startModeParser =
+    option startModeReader $
+    long "mode" <> value Automatic <> helpDoc (Just helpText)
```

Comment:
Because of the fact that you're using `neat-interpolation` now you don't need to use `helpDoc` function and convert `Text` to `Doc`. Just use [`help` function](https://www.stackage.org/haddock/lts-10.0/optparse-applicative-0.14.0.0/Options-Applicative.html#v:help). It will be just `<> help helpText` where `helpText :: String; helpText = toString [N.text|...|]`